yb-voyager/src/tgtdb/yugabytedb.go

```diff
@@ -1329,3 +1353,26 @@ type NodeMetrics struct {
 	Status  string
 	Error   string
 }
+
+func (yb *TargetYugabyteDB) checkIfCurrentUserIsSuperUser() (bool, error) {
+	// SELECT rolname, rolsuper FROM pg_roles WHERE rolname='yugabyte';
+	query := "SELECT rolsuper FROM pg_roles WHERE rolname=current_user"
+	rows, err := yb.Query(query)
+	if err != nil {
+		return false, fmt.Errorf("querying if user is superuser: %w", err)
+	}
+	defer rows.Close()
```

Comment:
nit: log the closeErr

Reply:
> func (pgx.Rows) Close()
Close closes the rows, making the connection ready for use again. It is safe to call Close after rows is already closed.

It doesnt return an error